WebAssembly.Tests/TestUtility.cs

```diff
@@ -86,4 +86,23 @@ private static void AreNotEqual(object? a, object? b)
         if (b != null)
             Assert.IsNotNull(b.ToString());
     }
+
+    /// <summary>
+    /// Tests that an exception is thrown with the expected type and message.
+    /// </summary>
+    public static void ThrowsException<T>(string message, Action action)
```

Comment:
This is redundant with `Assert.ThrowsException`. That helper returns the caught exception for follow-up asserts and other checks.

Reply:
I've been using NUnit for years and never knew this! Perfect 👌